packages/mechanic-template/app/utils/mechanic.js

```diff
@@ -254,6 +254,10 @@ export class Mechanic {
   async done(el) {
     if (!this.settings.animated) {
       if (isSVG(el)) {
+        // This conditional is a patch to an error, needs to be revised:
+        if (!this.serializer) {
+          this.serializer = new XMLSerializer();
+        }
```

Comment:
Good point. I'm still not sure the `Mechanic` class structure is the best way to organize the shared mechanic functions, so let's discuss later today.